packages/web/components/SuccessPlayer.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Flex, Image, Text } from '@metafam/ds';
+import React from 'react';
+
+import AvatarImage from '../public/images/avatar.png';
+import { MetaLink } from './Link';
+
+export const SuccessPlayer: React.FC = () => {
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent="center"
+      flexDirection="column"
+      width="100%"
+      height="100%"
+    >
```

Comment:
If we are using these same props in every component, should extract this Flex into our own "PageContainer" component or something